interfaces/automated-testing/scd/scd.yaml

```diff
@@ -260,6 +260,9 @@ components:
         - Activated
         - Nonconforming
         - Contingent
+    ################################################################################
+    ####################### End of ASTM-standard definitions #######################
+    ################################################################################
     Priority:
```

Comment:
`Priority` is still an ASTM-specified data type so I don't think we want to override it here.  Plus, `Priority` is used in `OperationalIntentTestInjection` which will be specified for any USS implementing the ASTM standard, regardless of whether they're checking out for U-space.

Reply:
I agree